genai-perf/tests/test_retrievers/test_synthetic_prompt_generator.py

```diff
@@ -49,10 +53,30 @@ def test_synthetic_prompt_zero_token(self):
 
     def test_synthetic_prompt_nonzero_tokens(self):
         tokenizer = get_tokenizer(DEFAULT_TOKENIZER)
+        prompt_tokens = 123
+        tolerance = 2
         prompt = SyntheticPromptGenerator.create_synthetic_prompt(
             tokenizer=tokenizer,
-            prompt_tokens_mean=123,
+            prompt_tokens_mean=prompt_tokens,
             prompt_tokens_stddev=0,
         )
+        assert len(tokenizer.encode(prompt)) <= 123 + tolerance
+        assert len(tokenizer.encode(prompt)) >= 123 - tolerance
```

Comment:
boyscout update.
it failed once in all my testing.
i updated it to be a small range